billing-report-agent/src/main/java/com/epam/pipeline/billingreportagent/service/impl/converter/StorageToBillingRequestConverter.java

```diff
@@ -44,39 +47,28 @@
 import java.time.YearMonth;
 import java.util.Collections;
 import java.util.List;
+import java.util.Objects;
 import java.util.Optional;
 
 @Slf4j
 @SuppressWarnings("checkstyle:MagicNumber")
-public class AwsStorageToBillingRequestConverter implements EntityToBillingRequestConverter<AbstractDataStorage> {
+public class StorageToBillingRequestConverter implements EntityToBillingRequestConverter<AbstractDataStorage> {
 
-    public static final int BYTES_TO_GB = 1 << 30;
-    public static final int PRECISION = 5;
     private static final String STORAGE_SIZE_AGG_NAME = "sizeSumSearch";
     private static final String SIZE_FIELD = "size";
     private static final String REGION_FIELD = "storage_region";
-    private static final String ES_FILE_INDEX_PATTERN = "cp-%s-file-%d";
+    private static final String ES_FILE_INDEX_PATTERN = "*cp-%s-file-%d";
```

Comment:
I think it is better to put index pattern into application.properties.
Why do we need `*` in index pattern? As fas as I remember all indices have aliases `cp-$type-file-$id`.

Reply:
Extracted pattern to properties.

Indices contain a randomly generated string of 5 chars at the beginning.